src/components/styles/OlStyleFill.vue

```diff
@@ -13,6 +13,7 @@ import usePropsAsObjectProperties from "@/composables/usePropsAsObjectProperties
 const props = withDefaults(
   defineProps<{
     color?: string;
+    color2?: string;
```

Comment:
I like the general idea of allowing to define a gradient.
However, the current solution has two drawbacks I think:
- its limited to two colours
- the offset is fixed at 0.5

I suggest to not implemented it using `color2` but an explicit prop `gradient` like this:

```ts
type GradientColorStop = [number, string];
const gradient: GradientColorStop[] = [[0.2, "green"], [0.5, "red"], [0.9, "yellow"]] 
```

Whenever `gradient` is passed, the prop `color` is ignored and gradient is used.